src/System.Data.Odbc/src/System.Data.Odbc.csproj

```diff
@@ -155,7 +155,9 @@
     <Reference Include="System.Threading.ThreadPool" />
     <Reference Include="System.Threading.Timer" />
     <Reference Include="System.Transactions" />
+    <Reference Include="System.Transactions.Local" Condition="'$(TargetGroup)' == 'netcoreapp2.0'" />
```

Comment:
Why are the new references needed when they weren't needed before?

Reply:
Because we where building against netstandard and the type IDtcTransaction is part of netstandard.dll but for netcoreapp it was typeforwarded to System.Transactions.Local, so that is why for netcoreapp we need to add that reference. If not I would get:

```
 System\Data\Odbc\OdbcConnectionHandle.cs(273,104): error CS1069: The type name 'IDtcTransaction' could not be found in the namespace 'System.Transactions'. This type has been forwarded to assembly 'System.Transactions.Local, Version=0.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
' Consider adding a reference to that assembly. [D:\repos\corefx\src\System.Data.Odbc\src\System.Data.Odbc.csproj]
```

Reply:
Cool.